File: src/mitris/sim/core/simulation/api/SimulationClock.java
package mitris.sim.core.simulation.api;

/**
 *
 * @author jlrisco
 */
public class SimulationClock {

    protected double time;

    public SimulationClock(double time) {
        this.time = time;
    }

    public SimulationClock() {
        this(0);
    }

    public double getTime() {
        return time;
    }
    
    public void setTime(double time) {
        this.time = time;
    }
}

File: c++/src/xdevs/core/simulation/SimulationClock.cpp
/*
 * SimulationClock.cpp
 *
 *  Created on: 5 de may. de 2016
 *      Author: jlrisco
 */

#include "SimulationClock.h"

SimulationClock::SimulationClock() : time(0) { }

SimulationClock::SimulationClock(const double& time) : time(time) { }

SimulationClock::~SimulationClock() { }

double SimulationClock::getTime() { return time; }

void SimulationClock::setTime(const double& time) { this->time = time; }
